pyronear/datasets/wildfire/wildfire.py

```diff
@@ -5,6 +5,8 @@
 import numpy as np
 import pandas as pd
 import torch
+from random import SystemRandom
```

Comment:
When I use random I got this error
 `Standard pseudo-random generators are not suitable for security/cryptographic purposes.` 
in Codacy/PR Quality Review so I switch to SystemRandom

Reply:
I'm not familiar with `SystemRandom` but if we only need a scalar random value, you can easily discard Codacy warning. If there is indeed a lack of availability on some systems as mentioned in the earlier reference, it might be safer to stick with a bare `random.random()`.

Alternatively, this file has a numpy dependency which has a large support of `numpy.random` !